### PR TITLE
Update functions.php

### PR DESCRIPTION
**Pull request by gcoeugnet**
Changes related to this issue : https://github.com/librenms/librenms/issues/3090

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -580,6 +580,7 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         'port' => $port,
         'transport' => $transport,
         'status' => '1',
+        'ip' => $ip,
```
Comment (laf) [inline includes/functions.php]: From the looks of it, $ip will be just the IP itself. This needs converting using the function we use elsewhere.

Comment (gcoeugnet) [inline includes/functions.php]: Sorry, I don't get it. In the device table, 'ip' field represent indeed the ip of the device. What do you mean by 'converting', I should convert this ip into what ? 
$ip is filled and checked before being recorded in database, regarding the following code : 
```
...
 256         if ($config['addhost_alwayscheckip'] === TRUE) {
 257             $ip = gethostbyname($host);
 258         } else {
 259             $ip = $host;
 260         }
 261         if (ip_exists($ip) === false) {
...
```

Comment (laf) [inline includes/functions.php]: ip is a varbinary field:

| ip                        | varbinary(16)                                | NO   |     | NULL    |                |

Check poll_device() function:

$db_ip = inet_pton($ip);

Comment (gcoeugnet) [inline includes/functions.php]: Ok, I get it. I check this out.

**Comment by murrant** [comment]
Thanks, fixed in PR #3100

**Pull request closed without merging**